wysadzulice/tests.py

```diff
@@ -1,3 +1,11 @@
-from django.test import TestCase
+from django.test import TestCase, Client
```

Comment:
Ja wiem, że w pep8 piszą że taki import jest ok - i że linter jak wszystkie symbole będą użyte to nie będzie się czepiał.

Bardziej chodziło mi o sytuację, w której masz 10 importów z tego samego modułu:
```python
from jakis.modul import (jedno, drugie, trzecie,
    czwarte, piate)
```
i masz jeden symbol nieużyty - u mnie wtedypodświetla w Sublimie ```from jakis.modul import...``` i albo doczytam na dolnej belce co jest nie tak, a wolę, gdy podświetla mi jedną linię i ja od razu wiem, który symbol powoduje problem.

Takie lekkie zboczenie :)